artview/components/plot_radar.py

```diff
@@ -44,6 +44,7 @@ class RadarDisplay(Component):
     Vgatefilter = None  #: see :ref:`shared_variable`
     VplotAxes = None  #: see :ref:`shared_variable` (no internal use)
     VpathInteriorFunc = None  #: see :ref:`shared_variable` (no internal use)
+    VpyartDisplay = None  #: see :ref:`shared_variable`
```

Comment:
We first of all, it is already there, I have done this some time ago for the background, this line is only so that it appears in the docs (by the way it is been some time I have updates the docs)

second, this is not our display, but the pyart display class. I have never a problem with it, it was just redundant. Now it is needed because the display is what control the image coordinates. It also add the possibility on some future optimization where the radar values get changed but not the geometry.